Pages/index.html

```diff
@@ -13,89 +13,82 @@
     <script src="https://cdn.jsdelivr.net/npm/swiper@11/swiper-bundle.min.js"></script>
 </head>
 <body>
-    <section class="sectionOne">
-        <h1>Cynthonia(Tone)</h1>
-        <hr>
-        <nav class="nav">
-          <ul class="navLinks">
-            <li>About</li>
-            <li>Education</li>
-            <li>Previous Projects</li>
-            <li>Works in Progress</li>
-          </ul>
-        </nav>
-        <hr>
+  <article class="page">
+    <section>
+      <h1>Cynthonia(Tone)</h1>
+      <hr>
+      <nav>
+        <ul>
+          <li>About</li>
+          <li>Education</li>
+          <li>Previous Projects</li>
+          <li>Works in Progress</li>
+        </ul>
+      </nav>
+      <hr>
     </section>
 
-    <section class="sectionTwo">
-        <div class="row sectionTwodiv">
+    <section>
+      <h2>Get To Know A Bit About Me</h2>
+        <div class="row secTwo">
           <img src="/Images/tanBlazer.jpeg" alt="Profile image of Cynthonia sitting at a table, in a tan blazer." class="profileImage">
-          <h2>Get To Know A Bit About Me</h2>
-          <div class="row secTwopara">
+          <div class="secTwopara">
+            <div class="secTwoScroll">
             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
           </div>
         </div>
     </section>
-    <br><br><br>
-    <div class="secTwoimages">
```

Comment:
i took out the old image gallery, so we're only using the carousel. hope that makes sense!